waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -0,0 +1,76 @@
+## Nim-LibP2P
```

Comment:
Leftover copy paste? I don't think we have this part in other nim files

Maybe just mention it was taken/inspired by nim-libp2p with brief link?

Reply:
I'd also add a link to RFC

Reply:
Yes, my fault with copy paste from my bigger WIP source! No problem adding links in the source, although this part doesn't directly come from libp2p noise implementation.